tutorials/getting_started/predicting_paper_venues/predicting_paper_venues_pt2.md

```diff
@@ -199,10 +199,15 @@ you just need to specify an `--output-file`.
 ## Running a Web Demo
 
 Once you have a trained model and a predictor,
-it's easy to run a simple web demo:
+it's easy to run a simple web demo. First, install the code containing the server like so:
 
 ```
-$ python -m allennlp.service.server_simple --help
+git clone https://github.com/allenai/allennlp-server-misc.git
+pip install -r allennlp-server-misc/requirements.txt
```

Comment:
Why not call it `allennlp-server` instead of `allennlp-server-misc`?  I'm not sure the extra `-misc` gives us anything.

Reply:
I called it that because I thought it would be confusing that it _wasn't_ the code that runs our demo.

Reply:
But we have allennlp-demo. Seems easy to solve that confusion with a note on the top of the README, instead of needless additions to the repo name.

Reply:
sure, as you wish